src/phoenix/server/api/helpers/playground_registry.py

```diff
@@ -0,0 +1,64 @@
+from typing import TYPE_CHECKING, Any, Callable, Dict, List, Optional, Tuple, Type
+
+from phoenix.server.api.types.GenerativeProvider import GenerativeProviderKey
+
+if TYPE_CHECKING:
+    from phoenix.server.api.subscriptions import PlaygroundStreamingClient
+
+ModelKey = Tuple[GenerativeProviderKey, str | None]
```

Comment:
```suggestion
ModelName: TypeAlias = Optional[str]
ModelKey: TypeAlias = Tuple[GenerativeProviderKey, ModelName]
```

This type pops up throughout this file.